docs/operations/metrics.md

```diff
@@ -329,6 +329,7 @@ These metrics are for the Druid Coordinator and are reset each time the Coordina
 |`segment/size`|Total size of used segments in a data source. Emitted only for data sources to which at least one used segment belongs.|`dataSource`|Varies|
 |`segment/count`|Number of used segments belonging to a data source. Emitted only for data sources to which at least one used segment belongs.|`dataSource`|< max|
 |`segment/overShadowed/count`|Number of segments marked as unused due to being overshadowed.| |Varies|
+|`segment/danglingTombstone/count`|Number of dangling tombstones marked as unused.| |Varies|
```

Comment:
This does not seem consistent with the metric name in the code

Reply:
Good catch, updated docs